llvm/test/CodeGen/NVPTX/atomics-sm90.ll

```diff
@@ -1,142 +1,127 @@
 ; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
-; RUN: llc < %s -mtriple=nvptx -mcpu=sm_90 -mattr=+ptx78 | FileCheck %s --check-prefixes=CHECK
-; RUN: llc < %s -mtriple=nvptx64 -mcpu=sm_90 -mattr=+ptx78 | FileCheck %s --check-prefixes=CHECK64
-; RUN: llc < %s -mtriple=nvptx -mcpu=sm_86 -mattr=+ptx71 | FileCheck %s --check-prefixes=CHECKPTX71
```

Comment:
How do we know that `sm_86` doesn't support 32-bit compilation?

Reply:
Technically 32-bit compilation for sm_8x is supported with cuda 11.1-11.8.
It just about nobody cares about 32-bit compilation for the new GPUs. I figured that keeping this minor niche is just not worth the trouble.

On the other hand, resurrecting it is not a big deal. I've updated the patch.